### PR TITLE
Add concepts documentation

### PR DESCRIPTION
**Pull request by brendo**
When using this gem for the first time, my team and I struggled to learn the concepts that were involved, especially as we were coming from a "homegrown" solution to A/B testing that didn't use these terms.

To help, I wrote some concept documentation in our project. I thought I'd offer to backport it to the gem considering it's open-source and there may be others that would benefit.

If I've misunderstood anything, or things are wrong, please let me know!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Contributing
+
+1. Fork it
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Commit your changes, including tests (`git commit -am 'Added some feature'`)
+4. Push to the branch (`git push origin my-new-feature`)
+5. Create new Pull Request, and mention @wvanbergen.
```
Comment (wvanbergen) [inline CONTRIBUTING.md]: I am happy to let you maintain this thing. I have not worked in this space for a long long time :)

File: docs/concepts.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Verdict Concepts
+
+Understanding the following Verdict concepts can help implement experiments and discuss them with peers.
```
Comment (gmcgibbon) [inline docs/concepts.md]:
```suggestion
Understanding the following Verdict concepts can help you implement experiments and discuss them with peers.
```

File: docs/concepts.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- [Group](#group)
+- [Assignment](#assignment)
+
+### Experiment
+
+`Experiment`s are defined in `app/experiments` and encapsulate the details of the test including metadata such as the name, description or owners, as well as any [qualifiers](#qualifiers) or the [groups](#groups) available in the test.
+
+### Qualifers
+
+Qualifiers are methods that take two parameters, `subject` and `context` and returns `true` if the `subject` can take this experiment, or `false` if the `subject` is not eligible. Experiments can have multiple qualifiers which are evaluated in succession. If a qualifier returns `false` no further qualifiers are checked.
```
Comment (gmcgibbon) [inline docs/concepts.md]:
```suggestion
Qualifiers are methods that take two parameters, `subject` and `context`. They return `true` if the `subject` can take this experiment, or `false` if the `subject` is not eligible. Experiments can have multiple qualifiers which are evaluated in succession. If a qualifier returns `false`, no further qualifiers are checked.
```

File: docs/concepts.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Examples of qualifiers might be:
+
+- Include subjects that originate from SpeedCurve or WebPage Test
```
Comment (gmcgibbon) [inline docs/concepts.md]: I don't understand this point, can you clarify or point to a simpler example?

**Pull request merged**